PROTO_tests/tests/numeric_test.py

```diff
@@ -446,8 +457,12 @@ def test_value_counts(self, num_type):
 
     def test_value_counts_error(self):
         pda = ak.linspace(1, 10, 10)
-        with pytest.raises(TypeError):
-            ak.value_counts(pda)
+
+        # This first test should not raise TypeError, and so it causes the test to fail.
```

Comment:
This seems outdated.  Maybe the original couldn't handle floats?  I would recommend to just remove it.